common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -1478,13 +1478,15 @@ public void onComplete(String streamId) throws IOException {
         if (isTooLate(info, partitionInfo.reduceId)) {
           freeDeferredBufs();
           mergeManager.pushMergeMetrics.lateBlockPushes.mark();
+          updateIgnoredBlockBytes();
```

Comment:
This is not needed. There is an exception thrown in the next line which triggers onFailure